packages/flutter_driver/lib/src/driver/web_driver.dart

```diff
@@ -0,0 +1,242 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+import 'dart:math' as math;
+
+import 'package:matcher/matcher.dart';
+import 'package:meta/meta.dart';
+import 'package:webdriver/sync_io.dart' as sync_io;
+import 'package:webdriver/support/async.dart';
+
+import '../common/error.dart';
+import '../common/message.dart';
+import 'driver.dart';
+import 'timeline.dart';
+import 'web_driver_config.dart';
+
+export 'web_driver_config.dart';
+
+/// An implementation of the Flutter Driver using the WebDriver.
```

Comment:
Can you add the explanation on how to run this mode in a doc comment here?

Reply:
Essentially the sets to test from above

Reply:
Done